nvme-print-json.c

```diff
@@ -4640,6 +4640,21 @@ static void json_output_perror(const char *msg)
 	json_output_object(r);
 }
 
+static void json_output_key_value(bool space, bool eol, const char *key, const char *val,
```

Comment:
The two arguments `space` and `eol` should not be exposed to the user here. First, having function with several arguments using bools are always hard to read, secondly, this leaks the output layer features to the users side.